synapse/lib/stormtypes.py

```diff
@@ -2519,7 +2557,8 @@ class Prim(StormType):
     '''
     The base type for all Storm primitive values.
     '''
-    _storm_typename = None
+    _storm_typename: Any = None
```

Comment:
I am still unconvinced that adding this adds anything useful... I guess it's time to discuss again?

Reply:
Please remove this and the typing import so we can merge.  We can create a ticket to discuss this change in convention.